src/System.ComponentModel.Composition/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap;
+    </BuildConfigurations>
```

Comment:
I assume we plan to enable netfx testing at some point? Do we have a tracking issue for that?

Reply:
Yes, it will be as part of: https://github.com/dotnet/corefx/issues/24903 as that is why we can't add netfx testing because we're testing against the netstandard turd assembly.